app/models/answer.js

```diff
@@ -1,41 +1,43 @@
 /**
  * Module dependencies.
  */
-var mongoose = require('mongoose'),
-    config = require('../../config/config'),
-    Schema = mongoose.Schema;
+const mongoose = require('mongoose'),
+  Schema = mongoose.Schema;
 
 /**
  * Answer Schema
  */
-var AnswerSchema = new Schema({
-    id: {
-        type: Number
-    },
-    text: {
-        type: String,
-        default: '',
-        trim: true
-    },
-    official: {
-        type: Boolean
-    },
-    expansion: {
-        type: String,
-        default: '',
-        trim: true
-    }
+const AnswerSchema = new Schema({
+  id: {
+    type: Number
+  },
+  text: {
+    type: String,
+    default: '',
+    trim: true
+  },
+  official: {
+    type: Boolean
+  },
+  expansion: {
+    type: String,
+    default: '',
+    trim: true
+  },
+  location: {
+    type: String
+  }
 });
 
 /**
  * Statics
  */
 AnswerSchema.statics = {
-    load: function(id, cb) {
-        this.findOne({
-            id: id
-        }).select('-_id').exec(cb);
-    }
+  load(id, cb) {
```

Comment:
Thank you for noticing that, the variable in question, was named by the original creators of the game, and the same naming convention was used in multiple files, changing just this variable would result in a break in convention of an already existing convention(although a bad convention, its a naming-convention nonetheless) and changing the naming-convention for all the variables, in all files of this repo is too big a change to make.